third_party/terraform/tests/resource_logging_metric_test.go

```diff
@@ -51,3 +74,24 @@ resource "google_logging_metric" "logging_metric" {
 	}
 }`, suffix, filter)
 }
+
+func testAccLoggingMetric_explicitBucket(suffix string, filter string) string {
+	return fmt.Sprintf(`
+resource "google_logging_metric" "logging_metric" {
+  name = "my-custom-metric-%s"
+  filter = "%s"
+
+  metric_descriptor {
+    metric_kind = "DELTA"
+    value_type  = "DISTRIBUTION"
+  }
+
+  value_extractor = "EXTRACT(jsonPayload.metrics.running_jobs)"
+
+  bucket_options {
+    explicit_buckets {
+      bounds = [0,1,2,3,4]
```

Comment:
I think this looks good now but let's make one of these a decimal since the type is `double`.